presentation_parser.py

```diff
@@ -0,0 +1,117 @@
+from pptx import Presentation
+from typing import Dict
+
+
+class PresentationParser:
+    """A class for parsing text from PowerPoint presentations."""
+
+    def __init__(self, path_to_presentation) -> None:
+        """Initializes a new instance of the pptx_parser class.
+        Args:
+            path_to_presentation (str): The file path of the PowerPoint presentation to parse.
+        Raises:
+            FileNotFoundError: If the provided file path is not valid.
+        """
+        self.presentation = None
+        try:
+            self.presentation = Presentation(path_to_presentation)
+        except FileNotFoundError:
+            raise FileNotFoundError("Presentation path not found.")
+
+        # dict[slide number : text extracted from slide]
```

Comment:
Dont leave commented out code inside your codebase! This leads to a messy codebase